tests/trainer/test_trainer.py

```diff
@@ -1069,6 +1072,69 @@ def test_dataloader_without_dataset(self):
         trainer.train()
         trainer.evaluate()
 
+    @require_galore_torch
+    def test_galore(self):
```

Comment:
It would be really cool if we could record the memory usage to show that GaLore works correctly, but that's probably not easily possible, or is there something in transformers that enables this?

Reply:
Thanks ! I did something in these lines in one of the tests below!